src/components/organisms/About/About.jsx

```diff
@@ -6,19 +6,19 @@ const About = (props) => {
       <div className="about-image-container">
         <img
           className="about__image2"
-          src={props.data.about.aboutImages[0].path}
+          src={props.about.aboutImages[0].path}
           alt="about"
         />
         <img
           className="about__image1"
-          src={props.data.about.aboutImages[1].path}
+          src={props.about.aboutImages[1].path}
           alt="about"
         />
       </div>
       <div className="about-text-container">
-        <h1 className="heading">{props.data.about.title}</h1>
-        <h2 className="subheading">{props.data.about.subTitle}</h2>
-        <p className="about__paragraph">{props.data.about.text}</p>
+        <h1 className="heading">{props.about.title}</h1>
```

Comment:
When these titles transition, it looks ugly, it needs more work. 
Was it in the requirements to make them slide together with the images?

Reply:
Tried to changed it, please check